apps/filebrowser/src/filebrowser/views.py

```diff
@@ -520,6 +521,9 @@ def listdir_paged(request, path):
     filter=?          - Specify a substring filter to search for in
                         the filename field.
   """
+  if ENABLE_NEW_STORAGE_BROWSER.get():
+    return render('storage_browser.mako', request, {})
```

Comment:
Nice, this is needed to get the new react app started, but additional routing within the new storage browser should probably be handed by a browser based router like the react router. Lets discuss once we get there.